cloud_experts_tutorials/cloud-experts-consistent-egress-ip.adoc

```diff
@@ -239,18 +267,18 @@ spec:
     - 10.10.200.253/32
 EOF
 ----
-
-Retrieve the load balancer hostname as the `LOAD_BALANCER_HOSTNAME` environment variable which you can copy and use for following steps:
-
++
+. Retrieve the load balancer hostname and save it as an environment variable by running the following command:
++
 [source,terminal]
 ----
 $ export LOAD_BALANCER_HOSTNAME=$(oc get svc -n default demo-service -o json | jq -r '.status.loadBalancer.ingress[].hostname')
 ----
 
-=== Test namespace egress
-
-Test the namespace egress rule which was created previously. The following starts an interactive shell which allows you to run curl against the demo service:
+=== Testing the namespace egress
 
+. Start an interactive shell and run `curl` against the demo service to test the namespace egress rule:
```

Comment:
You use the `curl` command in the next step. Little confusing to introduce it here. Maybe explain the process before the steps?